src/index.js

```diff
@@ -0,0 +1,22 @@
+import './style.css';
+
+const list = [
+  {
+    index: 1,
+    completed: false,
+    Description: 'Morning fellowship',
+  },
+  {
+    index: 2,
+    completed: false,
+    Description: 'microverse morning session',
+  },
+  {
+    index: 3,
+    completed: false,
+    Description: 'Have breakfast',
+  },
+];
+document.getElementById('id1').innerHTML = list[0].Description;
+document.getElementById('id2').innerHTML = list[1].Description;
+document.getElementById('id3').innerHTML = list[2].Description;
```

Comment:
- [ ] Kindly make sure your application has a function that iterates over the tasks array and populates the HTML list item element for each task.

- [ ] Once you implemented the above change, you should ensure that  on page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.